src/metabase/task/analyze_queries.clj

```diff
@@ -0,0 +1,61 @@
+(ns metabase.task.analyze-queries
+  (:require
+   [clojurewerkz.quartzite.jobs :as jobs]
+   [clojurewerkz.quartzite.schedule.simple :as simple]
+   [clojurewerkz.quartzite.triggers :as triggers]
+   [metabase.public-settings :as public-settings]
+   [metabase.query-analysis :as query-analysis]
+   [metabase.task :as task]
+   [metabase.util :as u]
+   [metabase.util.log :as log])
+  (:import
+   (org.quartz DisallowConcurrentExecution)))
+
+(set! *warn-on-reflection* true)
+
+(def ^:private max-cpu-usage-ratio 0.2)
+
+(def ^:private wait-ratio
+  (/ (- 1 max-cpu-usage-ratio) max-cpu-usage-ratio))
```

Comment:
For the benefit of any bozos looking at this:

![image](https://github.com/user-attachments/assets/0c6e01c8-d633-4bbc-ae56-d807e1100501)

Reply:
(it's me. I'm the bozo)

Reply:
Thanks! Maybe we could capture the math in a comment or unit test.

It feels weird testing an equation though.